src/Components/SingleRoom/SingleRoom.css

```diff
@@ -30,7 +30,7 @@ body {
 
 .container-p2 {
     bottom: 48vh;
-    left: 18vw;
+    left: 13vw;
```

Comment:
please use relative/absolute positioning to position containers/players in correct places.